resources/public/api/conf/1.0/schemas/retrieve_crystallisation_obligations_response.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Retrieve Income Tax Crystallisation Obligations",
+  "description": "An array to return one or more businesses and their obligations details",
+  "type": "object",
+  "properties": {
+    "obligationDetails": {
```

Comment:
Still need to confirm from Francis whether this is "obligationDetails" or "obligations"